Prepare release of v6.0 (#126)
quantum-journal/quantum-journal: README.md

> In addition, this document class come bundled with two new extras (currently in beta phase):

Reviewer: ```suggestion
In addition, this document class comes bundled with two new extras (currently in beta phase):
```

> 1. The quantum-lyx-template.lyx LxY layout, which allow you to generate the LaTeX source of your quantumarticle manuscript with the LyX document processor.

Reviewer: ```suggestion
1. The quantum-lyx-template.lyx LyX layout, which allows you to generate the LaTeX source of your quantumarticle manuscript with the LyX document processor.
```

2. The quantum.bst bibliography style for BibTeX, see the corresponding demo for usage details.

## Changelog

### New in v6.0

> * introduced the Quantum bibstyle `quantum.bst` together with the demo file `quantum_bst_demo`

Author: Can we please rename that to quantum-bibliographystyle-demo.tex

Reviewer: Done.

## Copyright

> Copyright 2017, 2018, 2019, 2020, 2021

Author: ```suggestion
Copyright 2017, 2018, 2019, 2020, 2021, 2022
```
